packaging/Shell/install-fedora-deps.sh

```diff
@@ -22,18 +22,18 @@ dnf -y install \
 
 # Dependencies
 dnf --refresh -y install \
-	python3-devel \
-	eigen3-devel \
+    python3-devel \
+    eigen3-devel \
     libxml2-devel \
-	spdlog-devel \
-	graphviz-devel \
-	sundials-devel \
-	gsl-devel
+    spdlog-devel \
+    graphviz-devel \
+    sundials-devel \
+    gsl-devel
 
 # Python
-pip3 install -U setuptools
-pip3 install -U wheel
-pip3 install -r requirements.txt
+pip3 install --upgrade setuptools
+pip3 install --upgrade wheel
+pip3 install .
```

Comment:
Any particular reason for the difference here about only `pip3 install .` vs `.[test,docs]`  in the `packaging/Shell/install-fedora-deps-dev.sh`

Reply:
@stv0g Doesn't this install the dpsim package as well and not just deps?